addons/website/static/src/js/widgets/theme.js

```diff
@@ -92,7 +92,11 @@ var QuickEdit = Widget.extend({
     /**
      * @private
      */
-    _onSaveClick: function () {
+    _onFocusLose: function (ev) {
+        if ($(ev.relatedTarget).is('.btn-secondary.fa-undo')) {
+            // when we click reset button
+            this._onResetClick();
```

Comment:
This is not the right way to handle this. If you only focus the reset button without clicking it, you don't want to save just because the input was focused out. The correct way would be to check the focus out of the whole quick edit widget.